src/views/Agent/Donate.js

```diff
@@ -0,0 +1 @@
+import React from './node_modules/react';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing file extension for "./node_modules/react"](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5592511)